app/src/test/shared/components/UploadFormTests.tsx

```diff
@@ -5,11 +5,16 @@ import {Sandbox} from "../../Sandbox";
 import {UploadFileForm} from "../../../main/shared/components/UploadFileForm";
 import {helpers} from "../../../main/shared/Helpers";
 import {Alert} from "reactstrap";
+import {CustomValidationResult} from "../../../main/shared/validation/FileValidationHelpers";
+import {mockFetcher, resetFetcher} from "../../mocks/mockRemote";
+import fetcher from "../../../main/shared/sources/Fetcher";
+import {ReportingFetcher} from "../../../main/report/sources/ReportingFetcher";
 
 describe('UploadForm', () => {
     let rendered: ShallowWrapper<any, any>;
     const sandbox = new Sandbox();
 
+    before(() => resetFetcher());
```

Comment:
Apparently which ever test is interfering with these tests wasn't using sandbox, so I also ended up having to put this here. If we ever track down the test that's polluting the state, this can be removed again. In the mean time, there's no harm in this getting run more often that is needed.